web/README.md

```diff
@@ -0,0 +1,59 @@
+# User Overrides Web Panel
+Communicate with the vehicle's CAN bus using a web application. Send custom or preset
+messages through the interactive and user-friendly panel.
+
+## Requirements
+The web panel is ran through the Flask web framework, which comes bundled to
+the default Python installation.
+
+In addition, the application depends on several web sockets to communicate
+with the server. Install them using:
+
+```bash
+pip install flask-socketio
+pip install gevent-websocket
+```
+
+Version inter-compatibility is highly limited, which may lead to several difficulties
+while running the web application. Visit the [version compatibility](https://flask-socketio.readthedocs.io/en/latest/#version-compatibility)
+if encountering such errors.
+
+The application uses [can_data/CAN2.dbc](https://github.com/msfrt/PiLogger/blob/web/web/can_data/CAN2.dbc)
```

Comment:
This link will not work one we merge and close the `web` branch. Since this we application is specifically for SR-20, you can link to the SR-20 DBC here: https://github.com/msfrt/Electrical-SR20/tree/master/DBCs

^it will always be in that link, since it's the master branch.